android/app/src/main/java/com/integreat/fetcher/FetchResult.java

```diff
@@ -1,16 +1,16 @@
 package com.integreat.fetcher;
 
-import java.time.ZonedDateTime;
+import java.util.Date;
```

Comment:
why did you revert this to date?

Reply:
ZonedDateTime depends on java 8 which is only available above api level 26